admin/tools/docker/latest/Dockerfile

```diff
@@ -1,40 +1,19 @@
-FROM debian:jessie
-MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
+FROM lsstsqre/centos:7-stackbase-devtoolset-6
```

Comment:
What's special about this version of the base container, an why not to use the standard one like centos/devtoolset-6-toolchain-centos7? Does the LSST one comes with extra RPMs required by the LSST Stack?

Reply:
Yes — devtoolset six and all platform-specific prereqs are cooked into this container.  It is maintained by SQuaRE as stack dependencies/tooling evolve, and is the official base container for all the non-qserv Jenkins container builds.

The hope is by using the same base container as the stack folk, we avoid inadvertent compiler-specific breakage as happened with the sphgeom thing recently.  I think this will be important until Qserv and stack/eups part ways.